data/map.txt

```diff
@@ -29124,6 +29246,35 @@ planet "New Switzerland"
 		threshold 2000
 		fleet "Small Republic" 11
 
+planet "New Tortuga"
+	attributes pirate "core pirate" core religious factory
+	landscape land/fog6
+	description `A planet of violent storms and frequent earthquakes, New Tortuga would turn away anyone unless they have a reason to hide. The few settlements on the planet are subterranean cave cities far from any fault lines. Thick blast doors block all entrances, and vast pumping systems remove water after the frequent supertyphoon-strength tropical storms.`
+	description `Most of the inhabitants of this planet are Syndicate dissidents who support themselves through manufacturing. The Syndicate often destroys pirate factories at their borders, but their fleets cannot penetrate as far as New Tortuga. Here, shipyards, factories, and outfitters provide an uninterrupted supply.`
+	spaceport `As you walk through the spaceport, you notice a surprising number of religious buildings scattered between the pirate bars and black market stores. Most of the shops are run not by pirates, but by ordinary folk, frequently wearing religious garb. They interact with pirates as customers and guests rather than as an oppressive government.`
+	spaceport `The relative peace is frequently broken by a pirate band wandering through the spaceport. The pirates here are generally more obedient and militarized than the ones seen on other planets. Their ships are more advanced too, judging by what sticks out their weapon and engine hardpoints.`
```

Comment:
Added some flavor here adding to the religious identity the pirates here seemed to be showing in the original text. Added some reverence to their behavior for all religions in the port. 

Alternative Spaceport (2) Description 1: "The relative peace is frequently broken by a pirate band wandering through the spaceport. While an auditory nuisance, the pirates here are generally more obedient and disciplined than the ones seen on other, perhaps less religious, planets. Their ships are more advanced too, judging by the armaments and engines you quickly noticed in the hanger."

Alternative Spaceport (2) Description 2: "The relative peace is frequently broken by a pirate band wandering through the spaceport. While perhaps an auditory nuisance, the pirates here are generally more obedient and disciplined than the ones seen on other, perhaps less religious, planets. The pirate gangs, maybe because of their faith and reverence to it, tend to settle down whenever they walk past a religious building, no matter the religion inside."